cineast-api/src/main/java/org/vitrivr/cineast/api/messages/query/Query.java

```diff
@@ -30,6 +33,7 @@ public Query(QueryConfig config) {
      *
      * @return
      */
+    @JsonIgnore //QueryConfig is handled by subclasses
     public final QueryConfig getQueryConfig() {
```

Comment:
This looks inconsistent to me, if the config is handled by the subclass, why does it have a field for it?